_posts/2019-11-27-cockpit-208.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 208
+author: pitti
+date: 2019-11-27
+tags: cockpit linux
+slug: cockpit-208
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
+
+Cockpit no longer offers the confusing "Default" choice when mounting a filesystem. Now every new filesystem requires the specification of a mount point. Additionally, Cockpit hides the distinction between configuration (`/etc/fstab`) and run-time state (`/proc/mounts`). Changes made in Cockpit always apply to both the configuration and run-time state. When configuration and run-time state differ from each other, Cockpit will warn about that and will let you bring them back in sync.
```

Comment:
I'm not sure what you're talking about here: the start of the sentence suggests that we will ignore the "mount point" flag stored in the filesystem itself, but then you say "every new filesystem requires the specification of a mount point".

What does this mean?  Do you mean that when creating a filesystem (ie: mke2fs) then mountpoint is required or when mounting a filesystem, mountpoint is required?

Reply:
It's the former, on fs creation you now always need to specify a mount point. I changed it a bit, is it clearer now? (I copied that from the PR from @mvollmer, maybe you also have a suggestion?)